src/cascadia/TerminalCore/ITerminalApi.hpp

```diff
@@ -40,13 +40,10 @@ namespace Microsoft::Terminal::Core
         virtual bool WarningBell() noexcept = 0;
         virtual bool SetWindowTitle(std::wstring_view title) noexcept = 0;
 
-        virtual bool SetColorTableEntry(const size_t tableIndex, const DWORD color) noexcept = 0;
+        virtual COLORREF GetColorTableEntry(const size_t tableIndex) const noexcept = 0;
+        virtual bool SetColorTableEntry(const size_t tableIndex, const COLORREF color) noexcept = 0;
 
         virtual bool SetCursorStyle(const ::Microsoft::Console::VirtualTerminal::DispatchTypes::CursorStyle cursorStyle) noexcept = 0;
-        virtual bool SetCursorColor(const DWORD color) noexcept = 0;
-
-        virtual bool SetDefaultForeground(const DWORD color) noexcept = 0;
-        virtual bool SetDefaultBackground(const DWORD color) noexcept = 0;

```

Comment:
I briefly thought "why not leave these in the header and just have them like instantly forward to the other one as a sort of `windowsx.h` shorthand so folks won't get too confused by having to pull a table index to adjust the defaults." but then I thought against it as it's really not that complicated to assume all colors are in the table.